ecosystem/sep-0031.md

```diff
@@ -181,12 +198,15 @@ The JSON object contains an entry for each asset that the anchor supports for re
 
 #### For each asset available for receiving, response contains:
 
-* `min_amount`: Optional minimum amount. No limit if not specified.
-* `max_amount`: Optional maximum amount. No limit if not specified.
-* `fee_fixed`: Optional fixed (flat) fee for deposit. In units of the received asset. Leave blank if there is no fee or the fee schedule is complex.
-* `fee_percent`: Optional percentage fee for deposit. In percentage points. Leave blank if there is no fee or the fee schedule is complex.
-* `sender_sep12_type`: Optional value of the `type` parameter the sending anchor should use for a `SEP-12 GET /customer` request. This field can be omitted if no KYC is necessary.
-* `receiver_sep12_type`: Optional value of the `type` parameter the sending anchor should use for a `SEP-12 GET /customer` request. This field can be omitted if no KYC is necessary.
+* `receiver_sep12_required`: a boolean value indicating whether or not the receiving anchor requires the recieving client to be registered via SEP-12. If `true`, the sending anchor should use the appropriate value from `sep12_types` in a [SEP-12 GET /customer?type=](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#customer-get) request to determine which fields should be collected.
+* `sender_sep12_required`: a boolean value indicating whether or not the receiving anchor requires the sending client to be registered via SEP-12. If `true`, the sending anchor should use the appropriate value from `sep12_types` in a [SEP-12 GET /customer?type=](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#customer-get) request to determine which fields should be collected.
+* `sep12_types`: (optional) an object containing the accepted values for the `type` parameter in [SEP-12](sep-0012.md) requests. Each key should map to an object value with a human-readable `description`. This field can be omitted from the response if no KYC is necessary for both the sender and receiver.
+* `min_amount`: (optional) minimum amount. No limit if not specified.
+* `max_amount`: (optional) maximum amount. No limit if not specified.
+* `fee_fixed`: (optional) fixed (flat) fee for deposit. In units of the received asset. Leave blank if there is no fee or the fee schedule is complex.
+* `fee_percent`: (optional) percentage fee for deposit. In percentage points. Leave blank if there is no fee or the fee schedule is complex.
+* `sender_sep12_type`: (**deprecated**, optional) the default value of the `type` parameter used in `SEP-12 GET /customer` requests for customers sending funds. This field can be omitted from the response if no KYC is necessary.
```

Comment:
❗ To say this field is the default value is to change the semantics of it a bit, which I don't think needs to happen. The field can have the description it had previously and you can add a statement that it is deprecated in favor of reading this other field that will list all possible types that can be used rather than one.